CONTRIBUTING.md

```diff
@@ -1,13 +1,20 @@
 Contributing
 -------------
 
-Before you contribute code to PHP\_CodeSniffer, please make sure it conforms to the PHPCS coding standard and that the PHP\_CodeSniffer unit tests still pass. The easiest way to contribute is to work on a checkout of the repository, or your own fork, rather than an installed PEAR version. If you do this, you can run the following commands to check if everything is ready to submit:
+Before you contribute code to PHP\_CodeSniffer, please make sure it conforms to the PHPCS
+coding standard and that the PHP\_CodeSniffer unit tests still pass. The easiest way to
+contribute is to work on a checkout of the repository, or your own fork, rather than an
+installed PEAR version. If you do this, you can run the following commands to check if
+everything is ready to submit:
 
     cd PHP_CodeSniffer
-    php scripts/phpcs
+    composer install
+    php bin/phpcs
 
-Which should give you no output, indicating that there are no coding standard errors. And then:
+Which should give you only informal output, indicating that there are no coding standard
+errors. And then:
 
-    phpunit
+    vendor/bin/phpunit
```

Comment:
If you already have PHPUnit installed, the command will be `phpunit` not some composer path to phpunit.